### PR TITLE
Update sound metadata api

### PR DESCRIPTION
**Pull request by vigneshka**

### CODE REVIEW

File: packages/indexer/src/metadata/extend/soundxyz/index.ts [hunk 1]
```diff
@@ -142,31 +168,33 @@ export const extendCollection = async (metadata: CollectionMetadata, _tokenId =
...
 
   if (!metadata.metadata) {
     metadata.metadata = {};
   }
 
-  metadata.metadata.imageUrl = releaseFromToken.coverImage.url;
-  metadata.metadata.description = releaseFromToken.description;
-  metadata.metadata.externalUrl = `https://sound.xyz/${releaseFromToken.artist.soundHandle}/${releaseFromToken.titleSlug}`;
+  metadata.metadata.imageUrl = release.coverImage.url;
+  metadata.metadata.description = release.behindTheMusic;
```
Comment (Raynos) [inline packages/indexer/src/metadata/extend/soundxyz/index.ts:191]: This field changed from `release.description` => `release.behindTheMusic` ; looks like a bug fix to me. :+1:

File: packages/indexer/src/metadata/providers/soundxyz-metadata-provider.ts [hunk 2]
```diff
@@ -131,8 +127,8 @@ export class SoundxyzMetadataProvider extends AbstractBaseMetadataProvider {
       community: "sound.xyz",
       metadata: {
         imageUrl: metadata.coverImage.url,
-        description: metadata.description,
-        externalUrl: `https://sound.xyz/${metadata.artist.soundHandle}/${metadata.titleSlug}`,
+        description: metadata.behindTheMusic,
```
Comment (Raynos) [inline packages/indexer/src/metadata/providers/soundxyz-metadata-provider.ts:130]: this again looks like a bug fix to me.

**Review by Raynos — COMMENTED** [review]
LGTM, Approved.

File: packages/indexer/src/metadata/extend/soundxyz/index.ts (hunk 1 above)
Comment (vigneshka) [reply packages/indexer/src/metadata/extend/soundxyz/index.ts:182]: good flag, seemed to have been working somehow before, unless this is a different field https://market.sound.xyz/1/0x37a03d4af1d7046d1126987b20117a0fdcbf6535/340

so def want to get additonal eyes here

**Review by ipeleg — APPROVED** [review]

**Pull request merged**